### ISSUE TITLE
Quantization Aware Training without Pruning for Yolov5

### ISSUE THREAD

**Issue by m-guise** [neuralmagic/sparseml#734]
The current sparseml supports pruning and pruning followed by quantization for Yolov5. Is there a way to use quantization aware training separately, without using pruning in the model. Can you provide a separate recipe or suggest some steps to do that?

Also, when can we expect sparseml support for Yolov5 Version 6.1, any estimated date or timeline?

**Comment by KSGulin** [comment]
@m-guise definitely, by modifying the recipe you can choose to run with only quantization aware training (QAT) or a number of other custom regimes. If you take for example the [YOLOv5s Pruned Quantized](https://sparsezoo.neuralmagic.com/models/cv%2Fdetection%2Fyolov5-s%2Fpytorch%2Fultralytics%2Fcoco%2Fpruned_quant-aggressive_94) model and look at the attached `original.md` recipe, it includes all the modifiers used to train this model. You'll see in there a modifier section titled `quantization_modifiers` with the modifier `QuantizationModifier` which drives QAT. Including this modifier in your recipe will add QAT to your training. You can read more on how to construct your own recipes [here](https://docs.neuralmagic.com/sparseml/source/recipes).

We updated our integration to support V6.1 earlier this month! Our research team is currently working on creating SOTA recipes for the updated architecture, but custom recipe support is in place if you'd like to apply a QAT recipe to your training. You'll find the updated model in the `model/` directory and the legacy model (version 5.0) in `models_v5.0/`

Let me know if you have any other questions on this front.

**Comment by m-guise** [comment]
Hi @KSGulin ,
It's been over a month, any updates when will the support for V6.1 become available?

**Comment by KSGulin** [comment]
Hi @m-guise, do you mean in terms of code support or SparseZoo models? Our integrations have included code support for V6.1 since April. On the SparseZoo model creation front we've had a shift in priorities on the research team and the YoloV5x6.1 models will take some more time to be deployed. 

You can still apply existing recipes (with minor modifications to the recipes in some cases) to train V6.1 models with SpraseML. Is there a specific recipe or sparsification target that you're aiming to use?

**Comment by m-guise** [comment]
Hi @KSGulin
I am looking for the recipes of Quantization Aware Training and Pruning (separately) for V6.1. If possible, can you please share these two? It will be very helpful.
